docs/guides/index.rst

```diff
@@ -0,0 +1 @@
+blah blah
```

Comment:
Is this file needed?

Reply:
No, sorry, that is a waste file, didn't remove earlier, I will remove.